### PR TITLE
Add Chevron

### PR DESCRIPTION
**Pull request by ramonimbao**
## Description

Add the Chevron: a through-hole 40% keyboard with rotary knob

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ramonimbao/chevron/config.h [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+//#define MATRIX_HAS_GHOST
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Small QoL improvements */
+#define PERMISSIVE_HOLD
+#define IGNORE_MOD_TAP_INTERRUPT
```
Comment (drashna) [inline keyboards/ramonimbao/chevron/config.h:95]: These are here twice. 
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/ramonimbao/chevron/config.h [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+//#define MAGIC_KEY_LAYER8         8
+//#define MAGIC_KEY_LAYER9         9
+//#define MAGIC_KEY_BOOTLOADER     B
+//#define MAGIC_KEY_BOOTLOADER_ALT ESC
+//#define MAGIC_KEY_LOCK           CAPS
+//#define MAGIC_KEY_EEPROM         E
+//#define MAGIC_KEY_EEPROM_CLEAR   BSPACE
+//#define MAGIC_KEY_NKRO           N
+//#define MAGIC_KEY_SLEEP_LED      Z
+
```
Comment (fauxpark) [inline keyboards/ramonimbao/chevron/config.h:172]:
```suggestion
```

File: keyboards/ramonimbao/chevron/config.h [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+   - MIDI notes can be added to the keymap
+   - Octave shift and transpose
+   - Virtual sustain, portamento, and modulation wheel
+   - etc.
+*/
+//#define MIDI_ADVANCED
+
+/* override number of MIDI tone keycodes (each octave adds 12 keycodes and allocates 12 bytes) */
+//#define MIDI_TONE_KEYCODE_OCTAVES 1
+
```
Comment (fauxpark) [inline keyboards/ramonimbao/chevron/config.h:212]:
```suggestion
```

File: keyboards/ramonimbao/chevron/readme.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32A
+
+To get to the bootloader,
+
+1. Press and hold the BOOT button
+
+2. Press the RESET button
+
+3. Release the BOOT button
```
Comment (fauxpark) [inline keyboards/ramonimbao/chevron/readme.md:16]:
```suggestion
 1. Press and hold the BOOT button
 2. Press the RESET button
 3. Release the BOOT button
```

File: keyboards/ramonimbao/chevron/readme.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Herringbone
```
Comment (fauxpark) [inline keyboards/ramonimbao/chevron/readme.md:1]:
```suggestion
# Chevron
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**